overlord/snapstate/snapstate.go

```diff
@@ -1371,6 +1371,7 @@ func finalizeUpdate(st *state.State, tasksets []*state.TaskSet, hasUpdates bool,
 		// re-refresh will check the lanes to decide what to
 		// _actually_ re-refresh, but it'll be a subset of updated
 		// (and equal to updated if nothing goes wrong)
+		sort.Strings(updated)
 		rerefresh := st.NewTask("check-rerefresh", fmt.Sprintf("Consider re-refresh of %s", strutil.Quoted(updated)))
```

Comment:
@degville the issue reported with this is that people thinks this is something asked of them.

So far our convention is that task summaries use the imperative.  But this one is kind of special anyway.

What we need to convey is a combination of checking/detecting + handling:

* Checking for re-refresh of %s
* Catch staged refresh of %s
* Handling staged refresh of %s

any input is appreciated

Reply:
@pedronis I can't easily think of an imperative construction without also implying user action. For me, _evaluate_ seems to work closest, maybe `Evaluate re-refresh of %s` (as it's an uncommon instruction), but it's still less ambiguous with something like `Re-refresh evaluation of %s`. I'll give this some more thought.

Reply:
as I tried to convey but wasn't very clear,  I'm fine to diverge and use a present participle here if it helps sort the confusion

Reply:
Ah, thanks. I can't think of anything better than _handling_ as it covers the checking part without always implying an action. I'd maybe add `status` (or _state_), eg: `Handling staged refresh and re-refresh status of %s`

Reply:
that looks fine but is a bit too long

Reply:
Maybe `Handling staged and re-refresh status of %s` is enough.

Reply:
Looks good with a small comment - perhaps replace with 'if' with 'as' because it will handle the re-refresh (somehow) even if not needed:

"Handling re-refresh of %s as needed."

Reply:
indeed, this sounds good.

Reply:
Updated.